docs/netlify.toml

```diff
@@ -141,3 +141,17 @@
     debug = true
     # Relative path to source directory in case you use Hugo's "--s" option
     srcdir = "docs"
+
+[[headers]]
+  #
+  # https://docs.netlify.com/configure-builds/file-based-configuration/#headers
+  #
+  for = "/*"
+  [headers.values]
+    Strict-Transport-Security = "max-age=31536000; includeSubDomains"
+    Content-Security-Policy = ""
+    X-Frame-Options = "SAMEORIGIN"
+    X-Content-Type-Options = "nosniff"
+    Referrer-Policy = ""
+    Permissions-Policy = ""
```

Comment:
this one is a tricky one. We need to understand if the application is using any of the functionality that need to be specifically allowed by Permission Policy. Try to use this one https://www.permissionspolicy.com/  to build a sample policy.

Reply:
use *, self or disabled

Reply:
Using disabled for now. We can update after we've decided the other rules are working right, as you said in your comment below...